chains/ethereum/src/utils.ts

```diff
@@ -18,16 +18,13 @@ export function walletConnectProvider({ projectId }: WalletConnectProviderOpts)
 }
 
 // -- connectors ------------------------------------------------------ //
-export function modalConnectors({ appName, chains: connectorChains }: ModalConnectorsOpts) {
```

Comment:
Could you explain why this type was removed? Isn't it a better pattern to keep ? Unless it was causing issues

Reply:
Not the type, change here is basically `chains: connectorChains` to just `chains` so its a bit cleaner